test/helpers.go

```diff
@@ -503,12 +504,12 @@ func registryClientOpts(ctx context.Context) []remote.Option {
 // setLocalEnv sets SIGSTORE_CT_LOG_PUBLIC_KEY_FILE, SIGSTORE_ROOT_FILE, and SIGSTORE_REKOR_PUBLIC_KEY for the locally running sigstore deployment.
 func setLocalEnv(t *testing.T, dir string) error {
 	// fulcio repo is downloaded to the user's home directory by e2e_test.sh
-	home, err := os.UserHomeDir()
-	if err != nil {
-		return fmt.Errorf("error getting home directory: %w", err)
+	home := os.Getenv(cloneDirEnvKey) //nolint:forbidigo
+	if home == "" {
+		t.Fatalf("clone directory env key not set: %s", cloneDirEnvKey)
 	}
```

Comment:
I think it would make a lot of sense to just embed the fulcio/config/ctfepubkey.pem test key here in the source file

Reply:
+1, if this is just to access the CT pub key, let's just bake it in, it'll simplify the test setup here and in e2e_test.go

Reply:
embedded